src/servers/trigger.ts

```diff
@@ -137,6 +142,39 @@ export async function onchainTransactionEventHandler(tx) {
   }
 }
 
+export async function onchainBlockEventhandler({ lnd, height }) {
+  // get incoming txs from the last 36 blocks ~6 hours
+  // we don't use default look back because daily cron job will take care of delayed transactions
+  const lookBack = 6 * 6 // hours * ~ blocks per hour
```

Comment:
is it ok to only check last 6 hours?

Reply:
why not just do the last block? (not sure how lnd is handling a reorg, maybe last 2 blocks?)

Reply:
I used onchain confirmations required + 1